zerver/outgoing_webhooks/__init__.py

```diff
@@ -1,8 +1,27 @@
 from __future__ import absolute_import
 from typing import Any, Dict, Text
 
-AVAILABLE_OUTGOING_WEBHOOK_INTERFACES = {}   # type: Dict[Text, Any]
+from zerver.outgoing_webhooks.generic import Generic
+from zerver.models import GENERIC_INTERFACE, Service
+
+AVAILABLE_OUTGOING_WEBHOOK_INTERFACES = {
+    GENERIC_INTERFACE: Generic
+}   # type: Dict[Text, Any]
 
 def get_service_interface_class(interface):
     # type: (Text) -> Any
-    return AVAILABLE_OUTGOING_WEBHOOK_INTERFACES[interface]
+    if interface is None or interface not in AVAILABLE_OUTGOING_WEBHOOK_INTERFACES:
+        return AVAILABLE_OUTGOING_WEBHOOK_INTERFACES[GENERIC_INTERFACE]
+    else:
+        return AVAILABLE_OUTGOING_WEBHOOK_INTERFACES[interface]
+
+def get_outgoing_webhook_service_handler(service):
+    # type: (Service) -> Any
+    # Load this lazily to prevent circular dependency problems
```

Comment:
Oops. sorry I forgot to remove this comment after I removed the import statement.